packages/db/tests/models/test-base-model.js

```diff
@@ -137,3 +146,47 @@ test('BasePgModel.exists() correctly returns false', async (t) => {
   const info = cryptoRandomString({ length: 5 });
   t.false(await basePgModel.exists(knex, { info }));
 });
+
+test('BasePgModel.delete() correctly deletes a record', async (t) => {
```

Comment:
Nit: if the intention is to allow delete*s* we should probably unit test a searchable query string. 

I grok that's more or less testing postgres, perhaps, but without doc strings in the base model and/or the possibility for adding some sort of check later, it seems a valid use case for the method. 

....or thinking about it, perhaps the nit is with the test title, which should be `correctly deletes records`.

Reply:
In either case, the test is centered on the `cumulus_id`, but can in theory take any partial.

Reply:
Yeah, that's fair. Here are a couple changes to hopefully clarify. 

- This changes the title of the test: 492ac3327560ab1c13ee9439d5e78d4c257bca91
- This adds docs to all of the methods in BasePgModel: a1ebc69a1c42e567195f2db8e388887e69af0ec4

Reply:
I'm not sure that entirely addresses the concern.     Delete takes:

```
   * @param {Partial<RecordType>} params - An object or any portion of an object of type RecordType

```
Which allows for multiple deletes, but the test with the updated title doesn't explicitly test that it deletes multiple records.   We should probably insert a couple of records, and then see that the partial deletes all of them based on the query string.   Should just require another insert statement.  That adequately tests the base case as well, in theory.    

Again, this is super nit territory as it strays into the testing the underlying postgres module, but it does guard against (unlikely) changes there.

Reply:
Oh! I think I totally misunderstood what you were saying. Try this. We're inserting 2 records with the same `{ info }` body. We call `basePgModel.delete(knex, { info })` and expect that both of those records were deleted.

5894548a632da3c8f058b3df6e760b0eb3b1ac7a

Reply:
👍🏻  that looks good, thanks!